tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -551,3 +557,713 @@ def activate_svc(client, service):
+                assert address in dns_response
+
+
+@pytest.fixture(scope='session')
+def rancher_compose_container(client, request):
+    if rancher_compose_con["container"] is not None:
+        return
+    cmd1 = \
+        "wget https://releases.rancher.com/compose/latest/" \
+        "rancher-compose-linux-amd64.tar.gz"
+    cmd2 = "tar xvf rancher-compose-linux-amd64.tar.gz"
+
+    hosts = client.list_host(kind='docker', removed_null=True)
+    assert len(hosts) > 0
+    host = hosts[0]
+    port = rancher_compose_con["port"]
+    c = client.create_container(name="rancher-compose-client",
+                                networkMode=MANAGED_NETWORK,
+                                imageUuid="docker:sangeetha/testclient",
+                                ports=[port+":22/tcp"],
+                                requestedHostId=host.id
+                                )
+
+    c = client.wait_success(c, 120)
+    assert c.state == "running"
+    time.sleep(5)
+
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(host.ipAddresses()[0].address, username="root",
+                password="root", port=int(port))
+    cmd = cmd1+";"+cmd2
+    print cmd
+    stdin, stdout, stderr = ssh.exec_command(cmd)
+    response = stdout.readlines()
+    found = False
+    for resp in response:
+        if "/rancher-compose" in resp:
+            found = True
+    assert found
+    rancher_compose_con["container"] = c
+    rancher_compose_con["host"] = host
+
+    def remove_rancher_compose_container():
+        delete_all(client, [rancher_compose_con["container"]])
+
+    request.addfinalizer(remove_rancher_compose_container)
+
+
+def launch_rancher_compose(client, env, testname):
+    compose_configs = env.exportconfig()
+    docker_compose = compose_configs["dockerComposeConfig"]
+    rancher_compose = compose_configs["rancherComposeConfig"]
+
+    access_key = client._access_key
+    secret_key = client._secret_key
+    docker_filename = testname + "-docker-compose.yml"
+    rancher_filename = testname + "-rancher-compose.yml"
+    project_name = env.name + "rancher"
+
+    cmd1 = "export RANCHER_URL=" + cattle_url()
+    cmd2 = "export RANCHER_ACCESS_KEY=" + access_key
+    cmd3 = "export RANCHER_SECRET_KEY=" + secret_key
+    cmd4 = "cd rancher-compose-v*"
+    cmd5 = "echo '" + docker_compose + "' > " + docker_filename
+    cmd6 = "echo '" + rancher_compose + "' > " + rancher_filename
+    cmd7 = "./rancher-compose -p " + project_name + " -f " + docker_filename + \
+           " -r " + rancher_filename + " up -d"
+
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(
+        rancher_compose_con["host"].ipAddresses()[0].address, username="root",
+        password="root", port=int(rancher_compose_con["port"]))
+    cmd = cmd1+";"+cmd2+";"+cmd3+";"+cmd4+";"+cmd5+";"+cmd6+";"+cmd7
+    print cmd
```

Comment:
print shows up in stdout only for failed test cases. Wanted to track the exact rancher-compose command that was executed and the response that was received for failed test cases.